doc/devel/documenting_mpl.rst

```diff
@@ -19,6 +24,30 @@ The requirements are as follows (https://github.com/matplotlib/matplotlib/blob/m
 6. pillow
 7. graphviz
 
+.. note::
+
+  * You'll need a minimal working latex distribution for many examples to run.
+  * Graphviz is not pip-installable so you need to install this on your own.
```

Comment:
externel link to Graphviz? (Graphviz is not a python package, so it needs to be installed separately.)